bindings/experimental/distrdf/python/DistRDF/Backends/Utils.py

```diff
@@ -134,25 +136,93 @@ def check_pcm_in_library_path(shared_library_path: str) -> Tuple[Set[str], Set[s
         list, list: Two lists, the first with all paths to pcm files, the
             second with all paths to shared libraries.
     """
+    
+    # issue when we have, for example ROOT.RDF.Experimental.Distributed.DistributeSharedLibs("folder/helpers_h.so") - this is False and should be True 
+    # therefore this will not work either and instead we check for the file endings of possible shared libraries 
+    # if os.path.isfile(shared_library_path):
+    #     shared_library_dir = os.path.dirname(os.path.abspath(shared_library_path))
+    #     #shared_library_dir = os.path.abspath(shared_library_path)
+    #     #+ "/"
```

Comment:
This comment is not completely clear to me, but it does not affect the rest of the function IIUC. Should we keep it, maybe rephrased?

Reply:
I think this comment can be removed - it was more an explanation for me for why this had to be changed to the way it is done now vs how it was done before some tests.